aries_cloudagent/protocols/present_proof/dif/pres_spec_schema.py

```diff
@@ -0,0 +1,24 @@
+"""DIF Proof Request Schema."""
+from marshmallow import fields
+
+from ....messaging.models.openapi import OpenAPISchema
+
+from .pres_exch import PresentationDefinitionSchema
+
+
+class DIFPresSpecSpecSchema(OpenAPISchema):
```

Comment:
SpecSpec? Is it a specification of a specification? Perhaps DIFPresSpecSchema might do?

Reply:
Renamed to `DIFPresSpecSchema` and moved to `present_proof/dif/pres_request_schema.py`